pynsee/macrodata/_get_dataset_list_internal.py

```diff
@@ -2,18 +2,19 @@
 import pkg_resources
 import pandas as pd
 
-from pynsee.utils._create_insee_folder import _create_insee_folder
-
+from pynsee.utils.save_df import save_df
+from pynsee.utils._get_temp_dir import _get_temp_dir
 
+@save_df(day_lapse_max=90)
 def _get_dataset_list_internal():
 
     zip_file = pkg_resources.resource_stream(__name__, "data/dataset_list_internal.zip")
 
-    insee_folder = _create_insee_folder()
-    dataset_file = insee_folder + "/" + "dataset_list_internal.csv"
+    temp_folder = _get_temp_dir()
+    dataset_file = temp_folder + "/" + "dataset_list_internal.csv"
```

Comment:
use ``join`` with ``from os.path import join``

Reply:
b27d6751ff1dc1738b522be9dce0680fd43e41d7 should tackle the issue